GameOfLife/game.cpp

```diff
@@ -1,38 +1,77 @@
 #include "Game.h"
 
-Board board(20,50);
+Board * board;
 
 Game::Game()
 {
-counterOfIterations = 0;
-filler.fillBoard(board);
-startInfiniteLoop();
+
+    counterOfIterations = 0;
+    int yourChoice;
+    do{
+
+        std::cout << "What you want to do:\n";
+        std::cout << "1. Load board form file txt\n";
+        std::cout << "2. Load board form file image\n";
+        std::cout << "3. Generating random board\n";
+        std::cout << "0. Exit\n";
+        std::cin >> yourChoice;
+        switch( yourChoice )  {
+        case 1:
+            std::cout<<"Type file name to load: ";
+            std::cin>>fileNameToLoad;
+            readerFromTxtFile.readFromGivenFile(fileNameToLoad);
+            if(validator.validate(readerFromTxtFile.getReadString()))
+            {
+                board = new Board(validator.getSizeOfBoard());
+                fillerWithValuesFromTxtFile.fillBoardWithGivenString(*board,readerFromTxtFile.getReadString());
+                startInfiniteLoop();
+                break;
+            }
+            else
+            {
+                display.displayErrorsCommand(validator.getErrorMessage());
+                break;
+            }
+        case 2:
+            //load board from image method
+            break;
+        case 3:
+            board = new Board(20,50);
+            filler.fillBoard(*board);
+            startInfiniteLoop();
+            break;
+        case 0:
+            std::cout <<"Goodbye.\n";
+            exit(1);
+            break;
+        }
+    }while (yourChoice != 0);
```

Comment:
This is temporary solution to add our new module to the program. This part of the code will be rebuild at the next sprint.